protocol/agents.json

```diff
@@ -223,6 +231,7 @@
         "php": "ably-php/${version}",
         "php-laravel": "ably-php/${version} laravel",
         "python": "ably-python/${version}",
+        "python-rest": "ably-python/${version} ably-python-rest",
```

Comment:
The entries in `ablyLibMappings` are used to map deprecated `X-Ably-Lib` headers to their `Ably-Agent` equivalent, so given ably-python has never set `python-rest` in the `X-Ably-Lib` header, this mapping is not needed.